modules/nf-core/khmer/trimlowabund/main.nf

```diff
@@ -0,0 +1,43 @@
+process KHMER_TRIMLOWABUND {
+    tag "${meta.id}"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/khmer:3.0.0a3--py37haa7609a_2' :
+        'biocontainers/khmer:3.0.0a3--py37haa7609a_2' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.abundtrim"), emit: sequence
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    trim-low-abund.py \\
+        -M ${task.memory.toGiga()}e9 \\
```

Comment:
There is some cases when task.memory might not be set, so you need to check for that. See the gatk or fgbio modules for instance.

Reply:
Ok, I have added this check in. Thanks!